### PR TITLE
Add staging fixes to master

### PR DESCRIPTION
**Pull request by samvrlewis**
# Description
I merged PRs for some basic fixes into the staging branch, but I think I misunderstood the purpose of staging and these probably should've been merged directly into master. Opening this now to get the changes into master now.

Referenced PRs:

https://github.com/swift-nav/libsbp/pull/1174
https://github.com/swift-nav/libsbp/pull/1175
https://github.com/swift-nav/libsbp/pull/1177
https://github.com/swift-nav/libsbp/pull/1178

# API compatibility

Does this change introduce a API compatibility risk?

No

# JIRA Reference

https://swift-nav.atlassian.net/browse/BOARD-XXXX

### CODE REVIEW

**Comment by samvrlewis** [comment]
Unless there's any problems in doing so, I will rebase and merge this (once approved) to avoid any conflicts between staging/master that would be caused by squashing

**Comment by samvrlewis** [comment]
If I'm understanding correctly, looks like this needs https://github.com/swift-nav/libsbp/pull/1179 to go in first - will wait for that to go in before merging.

**Review by silverjam — APPROVED** [review]

**Review by notoriaga — APPROVED** [review]

**Pull request merged**
